src/objective/cuda/cuda_regression_objective.cu

```diff
@@ -69,6 +69,126 @@ void CUDARegressionL2loss::LaunchGetGradientsKernel(const double* score, score_t
 }
 
 
+double CUDARegressionL1loss::LaunchCalcInitScoreKernel() const {
+  const double alpha = 0.9f;
```

Comment:
@shiyu1994 Why did you use `0.9` here while if I'm not mistaken original CPU implementation uses `0.5`?
https://github.com/microsoft/LightGBM/blob/c8712a9e083bfa0922f6e860f49a8a374816f43e/src/objective/regression_objective.hpp#L237